packaging_automation/common_tool_methods.py

```diff
@@ -78,6 +104,11 @@ def run(command, *args, **kwargs):
     return result
 
 
+def run_with_output(command, *args, **kwargs):
+    result = subprocess.run(command, *args, check=True, shell=True, stdout=subprocess.PIPE, **kwargs)
```

Comment:
As far as I know, using `shell=True` should be avoided when possible because this would mean you could execute a lot of unsafe things. Maybe see https://stackoverflow.com/questions/3172470/actual-meaning-of-shell-true-in-subprocess

Reply:
Fixed

Reply:
Looks like we still set `shell` to `True` by default, so this doesn't seem to be resolved yet.

Reply:
Sorry but my IDE deleted the code I wrote before I commit. :( I'm trying to solve that problem. Now I handled this again. Please check it